src/Sql/dbo/Stored Procedures/SsoConfig_Create.sql

```diff
@@ -0,0 +1,31 @@
+﻿CREATE PROCEDURE [dbo].[SsoConfig_Create]
+    @Id BIGINT OUTPUT,
+    @Enabled BIT,
+    @OrganizationId UNIQUEIDENTIFIER,
+    @Data NVARCHAR(MAX),
+    @CreationDate DATETIME2(7),
+    @RevisionDate DATETIME2(7)
+AS
+BEGIN
+    SET NOCOUNT ON
+    SET @Id = SCOPE_IDENTITY();
```

Comment:
Sorry, I should have specified... this statement should happen _after_ the `INSERT` so there's an identity value in scope.